configure.ac

```diff
@@ -207,7 +207,8 @@ AS_IF([test x"$enable_hardening" != x"no"], [
   AC_DEFUN([add_hardened_c_flag], [
     AX_CHECK_COMPILE_FLAG([$1],
       [EXTRA_CFLAGS="$EXTRA_CFLAGS $1"],
-      [AC_MSG_ERROR([Cannot enable $1, consider configuring with --disable-hardening])]
+      [AC_MSG_ERROR([Cannot enable $1, consider configuring with --disable-hardening])],
+      [-Werror]
```

Comment:
I think the reason Travis et al are failing, @williamcroberts , is because of this line. I just ran `gcc -Wbool` for grins, and it's an invalid flag, so with this change to the macro, it's now an error at configure-time, and therefore a show-stopper.